gossip/privdata/coordinator.go

```diff
@@ -824,6 +826,14 @@ func (c *coordinator) accessPolicyForCollection(chdr *common.ChannelHeader, name
 
 // isEligible checks if this peer is eligible for a given CollectionAccessPolicy
 func (c *coordinator) isEligible(ap privdata.CollectionAccessPolicy, namespace string, col string) bool {
+	// Simple check to see if mspID part of collection's MemberOrgs list
```

Comment:
isEligible is used upon commit to determine if the peer is eligible to retrieve/persist the private data collection data for a given transaction. May want to indicate this in the comment.